examples/bert/bert_hypertuning_config_classifier.py

```diff
@@ -0,0 +1,11 @@
+name = "bert_classifier"
+hidden_size = 768
+clas_strategy = "cls_time"
+dropout = 0.1
+num_classes = 2
+
+# hyperparams
+hyperparams = {
```

Comment:
Is it necessary to have this extra config file? To make the example look cleaner, can we just put `hyperparams` in `bert_with_hypertuning_main.py`, and reuse `config_classifier.py`?

Reply:
Resolved in fa829b6